scripts/google_research/open-covid-19/generate_csv_mcf.py

```diff
@@ -425,9 +425,7 @@ def generate_geo_mcfs(observation_abouts: pd.Series) -> List[str]:
     """Generates node MCFs for geos that do not come with DCIDs
     for local resolution."""
     mcfs = []
-    template = ('Node: {iso}\n'
-                'typeOf: schema:Place\n'
-                'isoCode: "{iso}"\n')
+    template = ('Node: {iso}\n' 'typeOf: schema:Place\n' 'isoCode: "{iso}"\n')
```

Comment:
This change seems to be breaking the lint tests.
I am guessing the latest version of yapf breaks strings into multiple lines while the earlier lines don't.